test.js

```diff
@@ -0,0 +1,64 @@
+var fs = require('fs');
+var crypto = require('crypto');
+
+var filename = 'README.md';
+var entryRegex = /\*\s[a-zA-Z\s]*\s\[.*\]\(.*\)/g;
```

Comment:
This regex would also match strings like
``` 
*   Abinash   Meher   [abinashmeher999](...)
```
i.e. multiple spaces
Since we wouldn't want this I would suggest something like `\*\s([a-zA-Z]+\s)+\[.*\]\(.*\)` because it is easy to overlook spaces. Please check if mine is syntactically correct, I am not sure.